packages/deployments/contracts/contracts/core/connext/interfaces/IStableSwap.sol

```diff
@@ -40,7 +40,15 @@ interface IStableSwap {
   function swapExact(
     uint256 amountIn,
     address assetIn,
-    address assetOut
+    address assetOut,
+    uint256 minAmountOut
```

Comment:
Hmm, maybe we should rename this method to `swapExactIn` to distinguish it from `swapExactOut`? Thoughts?